src/attachment.rs

```diff
@@ -19,6 +19,26 @@ pub struct Attachment {
     /// Fields are defined as an array, and hashes contained within it will be
     /// displayed in a table inside the message attachment.
     pub fields: Option<Vec<Field>>,
+    /// Optional text that appears above the attachment block
+    pub author_name: Option<SlackText>,
+    /// Optional link to the author
+    pub author_link: Option<SlackText>,
```

Comment:
Should be `Option<String>` because we don't need to do any escaping in the URL. We should send that as is (e.g. `&` shouldn't be changed to `&amp;` in a URL).